src/intercooler.js

```diff
@@ -1732,8 +1734,11 @@ var Intercooler = Intercooler || (function() {
         if (historyData) {
           processICResponse(historyData["content"], getTargetForHistory($('body')), true);
           if (historyData["yOffset"]) {
-            window.scrollTo(0, historyData["yOffset"])
-          }
+            window.scrollTo(0, historyData["yOffset"]);
+           }
+           if (historyData["title"]) {
+				  document.title = historyData["title"];
```

Comment:
Please change indentation to spaces rather than tabs for consistency.

Reply:
Fixed.